doc/api/process.md

```diff
@@ -706,14 +706,15 @@ console.log(process.env.test);
 // => 1
 ```
 
-## process.emitWarning(warning[, name][, ctor])
+## process.emitWarning(warning[, type[, code]][, ctor])
```

Comment:
@sam-github My thoughts exactly. Supporting two call signatures is easy and enables easier future additons.